common/lib/common-definitions/api-report/common-definitions.api.md

```diff
@@ -196,15 +196,15 @@ export interface ILoggingError extends Error {
 }
 
 // @public
-export interface ITaggedTelemetryPropertyType {
+export interface ITaggedTelemetryPropertyType<TValue> {
```

Comment:
we should be very cautions with generics, and especially adding new generics, as they can make the api really hard to consume. i don't actually see how this would be usable. you should add a type test to validate it

Reply:
here is an example of a package that does specific testing for types: https://github.com/microsoft/FluidFramework/blob/main/packages/common/core-interfaces/src/test/types/fluidObjectTypes.ts

Reply:
Yeah let's not swap the generic-based type in in-place.  We can revisit when we get back to this point in the staging.